test/helpers/builder.js

```diff
@@ -57,8 +58,34 @@ export class Builder {
       writer.add(entry.cid, entry.offset)
     }
     writer.close()
-    // @ts-expect-error node web stream is not web stream
-    await this.#satnav.put(`${cid}/${cid}.car.idx`, readable)
+
+    // @ts-expect-error
+    const indexBytes = concat(await collect(readable))
+    await this.#satnav.put(`${cid}/${cid}.car.idx`, indexBytes)
```

Comment:
nit: do we really need to buffer it in memory now?

Reply:
if so, could be cool to state reason for this in comment

Reply:
Oh, no we don't need to here...I was just copy pasta 🤪